docs/guide/headless-cms.md

```diff
@@ -0,0 +1,177 @@
+# Using Apostrophe as a headless CMS
+
+Apostrophe is fully featured as a traditional CMS (with some non-traditional features). It is also immediately ready for use as a headless CMS. There are many scenarios where Apostrophe works well as a headless CMS, including:
+
+- We may want to use Apostrophe to serve webpages normally, but also use the website data in another context, such as a mobile app.
+- We may want to go fully headless with a front end like Gatsby or Gridsome while also allowing editors to use Apostrophe's in-context editing.
+- We may have an app that handles features such as e-commerce but need a CMS to create and pipe content into the app.
+
+Regardless of the reason or the layers involved there are some common concepts to understand.
+
+## Accessing data using the REST APIs
```

Comment:
Should we include a note that we don't have GraphQL support yet, but it's something we're considering for our roadmap? (We should probably actually prioritize this on our roadmap...)

Reply:
[You can let folks know to share how important this is to them on our roadmap.](https://portal.productboard.com/apostrophecms/1-product-roadmap/c/44-graphql-api)

Reply:
😬 I'm not sure how much we should be doing that in docs. I guess we could use a note block to point to product board for input.

Reply:
I've added that at the end.